appsdk/sdk.go

```diff
@@ -463,7 +469,7 @@ func (sdk *AppFunctionsSDK) initializeConfiguration() error {
 			Port:            configuration.Registry.Port,
 			Type:            configuration.Registry.Type,
 			Stem:            internal.ConfigRegistryStem,
-			CheckInterval:   "1s",
+			CheckInterval:   configuration.Service.CheckInterval,
```

Comment:
We need to validate configuration.Service.CheckInterval since it is no longer in our control.

Reply:
good catch. will do